exporter/exporterhelper/factory.go

```diff
@@ -72,7 +72,7 @@ func WithLogs(createLogsExporter CreateLogsExporter) FactoryOption {
 	}
 }
 
-// WithCustomUnmarshaler overrides the default "not available" CustomUnmarshaler.
+// WithCustomUnmarshaler makes the factory to also implement component.ConfigUnmarshaler.
```

Comment:
To be more consistent with the other comments:
```suggestion
// WithCustomUnmarshaler implements component.ConfigUnmarshaler.
```

(same for elsewhere in this PR)

Reply:
Done.